.github/workflows/ci.yaml

```diff
@@ -107,6 +107,18 @@ jobs:
       needs.changes.outputs.fendermint == 'true' ||
       github.ref == 'refs/heads/main'
 
+  fevm-contract-tests:
```

Comment:
Do you think this should be a precondition for publishing?

Reply:
Hey! @aakoshh that is a great idea! I think that it depends on how long the tests take to run. If they're down to 2 minutes I think we can run them more often than just before publishing.